uievents/textInput/smiley-manual.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>textInput: smiley (manual)</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<style>
+[contenteditable] { border: thin inset silver; }
+</style>
+<p>Type "🙂" into each text field below.</p>
+<input class=test-el>
+<textarea class=test-el></textarea>
+<div contenteditable=true class=test-el></div>
+<script>
+setup({ explicit_timeout: true });
```

Comment:
Oh, adding a manual test, unfortunately...

Reply:
Yes, as far as I know this isn't automatable in wpt.